x-pack/test/functional/apps/discover/reporting.ts

```diff
@@ -80,6 +80,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     describe('Generate CSV: new search', () => {
       before(async () => {
         await reportingAPI.initEcommerce();
+        await kibanaServer.uiSettings.update({ 'dateFormat:tz': 'UTC' });
```

Comment:
Looking great,  just one more thing, I'd like to understand. Why is this necessary to be added to this test? Asking because we've got the same test in serverless version

https://github.com/elastic/kibana/blob/921067c95b4dc46cbc67d4eaa03eb774a6d3a384/x-pack/test_serverless/functional/test_suites/common/discover/x_pack/reporting.ts#L90

Reply:
Asking because those tests should be deduplicated in the future, right? so they shouldn't diverge

Reply:
@kertal there is a reason for the difference between the stateful and the serverless version of this test. The stateful version uses `esArchiver.emptyKibanaIndex`, which also resets the `dateFormat:tz` setting (which is `UTC` by default in our UI tests), so in that case we have to manually set it back to UTC in the test. The serverless version of the test is using `kibanaServer.savedObjects.cleanStandardList` instead of `emptyKibanaIndex` and this doesn't reset Kibana settings, so there's no need to reapply the time zone in these test.

Reply:
Ah, thx I see! I think this should be mentioned as a comment, because when those tests are deduplicated, this difference might raise a question ... or is this obvious?

Reply:
Anyway, I'm approving this now, this was my last question, it might be worth a comment in the code, for clarity. But not a reason to block moving forward

Reply:
I think it's not directly obvious, so adding a comment before the `await kibanaServer.uiSettings.update({ 'dateFormat:tz': 'UTC' });` line makes sense, maybe some thing like
```
esArchiver.emptyKibanaIndex() above also resets the Kibana time zone setting, so we're re-applying it here
```

Reply:
yes, this makes sense 👍